umake/frameworks/baseinstaller.py

```diff
@@ -396,7 +398,8 @@ def decompress_and_install_done(self, result):
             UI.return_main_screen(status_code=1)
 
         self.post_install()
-
+        if self.desktop_filename:
```

Comment:
no, remember that more than one ide have "eclipse" as exec binary name? Maybe you're right, just create an @property in the base installer converting desktop filename to the property, (basically calling `self.desktop_filename.split('.')[0]`)